src/ASM/LR/ASMLRSpline.h

```diff
@@ -207,6 +207,11 @@ class ASMLRSpline : public ASMbase, public ASMunstruct
   std::shared_ptr<LR::LRSpline> projB;  //!< Pointer to spline object of the projection basis
   std::shared_ptr<LR::LRSpline> projB2; //!< Pointer to spline object of the secondary projection basis
   std::shared_ptr<LR::LRSpline> refB;   //!< Pointer to spline object of the refinement basis
+
+  ThreadGroups threadGroups; //!< Element groups for multi-threaded assembly
+  ThreadGroups projThreadGroups; //!< Element groups for multi-threaded assembly - projection basis
+  ThreadGroups proj2ThreadGroups; //!< Element groups for multi-threaded assembly - second projection basis
+
```

Comment:
Remove blank line L214.
With this change also the non-mixed classes gets a `proj2ThreadGroups` member. That is intentionally, or an affordable side effect only. Do the non-mixed classes ever need a second projection basis?